src/main/java/frc/utils/ctre/BusChain.java

```diff
@@ -7,56 +7,70 @@
 
 public enum BusChain {
 
-	ROBORIO("rio"),
+	ROBORIO("Rio"),
 	CANIVORE("CANivore");
 
+	private static final double MAX_CAN_UTILIZATION_PERCENT = 0.6;
+	private static final double MAX_RECEIVE_ERRORS = 0;
+	private static final double MAX_TRANSMIT_ERRORS = 0;
+	private static final String LOG_PATH_PREFIX = "Bus/";
+
 	private final String chainName;
+	private final String logPath;
+	private CANBus.CANBusStatus busStatus;
 
 	BusChain(String chainName) {
 		this.chainName = chainName;
+		this.logPath = LOG_PATH_PREFIX + this.chainName + "/";
+		this.busStatus = CANBus.getStatus(getChainName());
 	}
 
 	public String getChainName() {
 		return chainName;
 	}
 
-	void updateStatus() {
-		CANBus.CANBusStatus busStatus = CANBus.getStatus(getChainName());
-		logStatus(busStatus);
-		reportAlerts(busStatus);
+	public void updateStatus() {
+		busStatus = CANBus.getStatus(this.chainName);
```

Comment:
no need this.

Reply:
no need for the updateStatus?

Reply:
```suggestion
		busStatus = CANBus.getStatus(chainName);
```
this. not the function

Reply:
oh lmao ok